### ISSUE TITLE
Make keywords filtering / faceted search work like CKAN - simultaneous to lines 1-10

### ISSUE THREAD

**Issue by kalxas** [state-hiu/2C-GeoNode#51]

**Comment by Lenninlasd** [comment]
Limit to 10 and text filter:
![atcl6bd8pe](https://user-images.githubusercontent.com/7197750/31790282-915bb2e2-b4da-11e7-9fbe-0553bc509d8d.gif)

**Comment by kalxas** [comment]
Action items:
- [ ] Clean up the keyword list
- [ ] Drop the keyword filter tab

**Comment by faithrs** [comment]
@kalxas see my comment for #50

**Comment by kalxas** [comment]
@faithrs thanks

**Comment by faithrs** [comment]
We like the keywords search function (it seems to be working well). Keep the search box and populate the list only after you begin the search (first three letters is great).

**Comment by kalxas** [comment]
Since the title and description of this title was about keyword filtering and facets, I think this is done.
@faithrs can you please open a new ticket with the new requirements from your last comment?

**Comment by faithrs** [comment]
Sorry @kalxas - I missed your message on this. Thanks for opening a new ticket.

**Comment by kalxas** [comment]
no worries @faithrs 
Lets discuss those tickets in today's meeting.
